### ISSUE TITLE
"CPU rendering" doesn't work

### ISSUE THREAD

**Issue by zixiCat** [cornerstonejs/cornerstone3D-beta#510]
When I close hardware acceleration of browser, it will throw the following error. I remember it worked a few months ago.
![1679575519766](https://user-images.githubusercontent.com/47551787/227210583-4d16b5bb-6210-4676-9441-05730d1c3638.jpg)

**Comment by sedghi** [comment]
Seems like you want to render volumes in cpu mode? we don't have that right now unfortunatelly. Our stack viewport should work though

**Comment by zixiCat** [comment]
@sedghi No, because some file cannot be run using segmentation volume in integrated graphics, maybe it's a file problem or the file size is large(1.6Mb per frame). but when I switch to cpu mode(seems to use google swiftshader) and it works.

**Comment by zixiCat** [comment]
@sedghi Not now, thanks for your answer.
